tf/env/production/buckets.tf

```diff
@@ -1,5 +1,5 @@
 module "production-buckets" {
-  source = "git::ssh://git@github.com/wmde/wbaas-deploy//tf//modules/buckets/?ref=tf-module-buckets-0"
+  source = "git::ssh://git@github.com/wmde/wbaas-deploy//tf//modules/buckets/?ref=tf-module-buckets-3"
```

Comment:
I'm surprised to see that although versions 0, 1 and 2 of this module existed for a long time now (2 was tagged late 2022), this is still using version 0. Could using 3 here introduce unwanted changes?

Reply:
Diffing the versions seems to be ok https://github.com/wmde/wbaas-deploy/compare/tf-module-buckets-0...tf-module-buckets-3?w=1 and not introduce any unwanted changes.